bin/restydoc-index

```diff
@@ -357,3 +358,28 @@ sub gen_aliases {
 
     return @aliases;
 }
+
+sub shell_quote() {
+    my $ret = shift;
+
+    if (!defined $ret or $ret eq '') {
+        return "''";
+    }
+
+    if ($ret =~ s/\x00//g) {
+        die "shell_quote(): No way to quote string containing null (\\000) bytes\n";
+    }
+
+    if ($ret =~ m|[^\w!%+,\-./:=@^]|) {
+        $ret =~ s/'/'\\''/g;
```

Comment:
It's easier and also safer to simply use `\` to escape everything instead of messing with single quotes or double quotes.